runtime/builtin_types_test.go

```diff
@@ -425,6 +425,72 @@ func TestBuiltinSetAttr(t *testing.T) {
 	}
 }
 
+func TestRawInput(t *testing.T) {
+	fun := newBuiltinFunction("TestRawInput", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+		// Create a fake Stdin for input test.
+		stdinFile, w, err := os.Pipe()
+		if err != nil {
+			return nil, f.RaiseType(RuntimeErrorType, fmt.Sprintf("failed to open pipe: %v", err))
+		}
+
+		go func() {
+			defer w.Close()
+			w.Write([]byte("hello grumpy\n"))
+			stdinFile.Sync()
+		}()
+
+		oldStdin := Stdin
+		Stdin = NewFileFromFD(stdinFile.Fd())
+		defer func() {
+			Stdin = oldStdin
+			stdinFile.Close()
+		}()
+
+		// Create a fake Stdout for output test.
+		r, stdoutFile, err := os.Pipe()
```

Comment:
@trotterdylan In this case, `builtinRawInput()` will return `*Object `and `*BaseException`, but `fn` in `captureStdout` want to return only `*BaseException`. It will works for any function I will pass?